packages/editor/src/browser/editor-manager.ts

```diff
@@ -277,10 +265,15 @@ export class EditorManager extends NavigatableWidgetOpenHandler<EditorWidget> {
         return this.open(uri, splitOptions);
     }
 
-    protected async revealSelection(widget: EditorWidget, input?: EditorOpenerOptions, uri?: URI): Promise<void> {
-        let inputSelection = input?.selection;
+    protected override async doOpen(widget: EditorWidget, uri: URI, options?: EditorOpenerOptions): Promise<void> {
+        await super.doOpen(widget, uri, options);
+        await this.revealSelection(widget, uri, options);
+    }
+
+    protected async revealSelection(widget: EditorWidget, uri: URI, options?: EditorOpenerOptions): Promise<void> {
```

Comment:
This is an unremarked breaking change that doesn't appear to actually be necessary, in the end - the logic in this function hasn't actually changed.